CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # mdn-bcd-collector Changelog
 
+## v6.0.3
+
+### Test Changes
+
+#### Changed
```

Comment:
@queengooborg looks like the rules to drop sections when empty isn't working here.

Reply:
Actually, they are, I just didn't think to drop the entire "Changed" section as well... 😅

Reply:
Right, there's more of the same logic that can be added.